### DISCUSSION TITLE
Add Generated Documentation Example to Repo

### DISCUSSION THREAD

**Discussion by thetechgy** [almenscorner/IntuneCD#44] [Ideas]
Is it possible to add an example of the generated documentation added to the repo, just so that we can see it without going through all of the setup steps?

**Comment by almenscorner** [comment]
Hi Travis,

Good idea, I'll look into adding it to the readme, in the mean time, here is a screenshot of the documentation:

![doc](https://user-images.githubusercontent.com/78877636/180316257-a3bb8c5f-163d-4bb4-a51e-ab1aa1af30ea.png)
